api/envoy/config/filter/accesslog/v2/accesslog.proto

```diff
@@ -73,6 +73,9 @@ message AccessLogFilter {
 
     // Response flag filter.
     ResponseFlagFilter response_flag_filter = 9;
+
+    // gRPC status filter.
+    GrpcStatusFilter grpc_status_filter = 10;
```

Comment:
Can you add a release note for this new feature (with a link to this field)?

Reply:
Does this just go in `//docs/root/intro/version_history.rst`?

Reply:
Yup. Take a look at other examples to get a feel for how we write them.

Reply:
Added in 7625146. Let me know if I should put it elsewhere.